core/src/main/scala/io/projectglow/transformers/normalizevariants/NormalizeVariantsTransformer.scala

```diff
@@ -94,21 +91,14 @@ class NormalizeVariantsTransformer extends DataFrameTransformer with HlsUsageLog
   }
 }
 
-object NormalizeVariantsTransformer extends HlsUsageLogging {
+object NormalizeVariantsTransformer extends HlsEventRecorder {
   val MODE_KEY = "mode"
   val MODE_NORMALIZE = "normalize"
   val MODE_SPLIT_NORMALIZE = "split_and_normalize"
   val MODE_SPLIT = "split"
   private val REFERENCE_GENOME_PATH = "reference_genome_path"
 
-  private[projectglow] def logNormalizeVariants(mode: String): Unit = {
-    val logOptions = Map(MODE_KEY -> mode)
-    recordHlsUsage(
-      HlsMetricDefinitions.EVENT_HLS_USAGE,
-      Map(
-        HlsTagDefinitions.TAG_EVENT_TYPE -> HlsTagValues.EVENT_NORMALIZE_VARIANTS
-      ),
-      blob = hlsJsonBuilder(logOptions)
-    )
+  def logNormalizeVariants(mode: String): Unit = {
```

Comment:
nit: Now that we have recordHlsEvent, I think it cleaner if we drop this function and plug recordHlsEvent with appropriate arguments in the the three cases above directly.